KubeArmor/core/kubeArmor.go

```diff
@@ -635,3 +640,37 @@ func KubeArmor() {
 	// destroy the daemon
 	dm.DestroyKubeArmorDaemon()
 }
+
+// StrToFile Function
+func KarmorDataToFile(val KarmorData, destFile string) {
```

Comment:
I think this whole function can be reduced to couple of lines. Something like this:

```go
j, err := json.Marshal(val)
// failure handling
err := ioutil.WriteFile(destFile, j, 0644)
// failure handlling
```

Why use `os.Stat()`, `os.Create()`, `os.Close()` etc?